src/ophys_etl/modules/suite2p_wrapper/schemas.py

```diff
@@ -230,13 +231,18 @@ def check_retain_files(self, data, **kwargs):
 
     @mm.post_load
     def copy_movie_frame_rate(self, data, **kwargs):
-        if data['movie_frame_rate'] is not None and \
-           data['movie_frame_rate_hz'] is None:
+        if not np.allclose(data['movie_frame_rate'],
```

Comment:
`np.allclose` won't know what to do if one of the args actually is `None`.

You need to enclose this in a
```
if movie_frame_rate is not None and movie_frame_rate_hz is not None
```
block

Reply:
Thanks for the catch. Just updated.